core/print_formatters.py

```diff
@@ -96,6 +96,23 @@ def print_schains(schains):
     print(Formatter().table(headers, rows))
 
 
+def print_dkg_statuses(statuses):
```

Comment:
I think we should add test for this function. We can just pass some test statuses and check stdout